docs/source/whatsnew_1_4.md

```diff
@@ -0,0 +1,59 @@
+# What's new in 1.4 🎉🎉
+
+- MAISI: state-of-the-art 3D Latent Diffusion Model
+- VISTA-3D: interactive foundation model for segmenting and anotating human anatomies
+- VISTA-2D: cell segmentation pipeline
+- Integrating MONAI Generative into MONAI core
+- Geometric Data Support
+
+
+## MAISI: state-of-the-art 3D Latent Diffusion Model
+
+![maisi](../images/maisi_train.png)
+
+NVIDIA MAISI (Medical AI for Synthetic Imaging) is a state-of-the-art three-dimensional (3D) Latent Diffusion Model designed for generating high-quality synthetic CT images with or without anatomical annotations. This AI model excels in data augmentation and creating realistic medical imaging data to supplement limited datasets due to privacy concerns or rare conditions. It can also significantly enhance the performance of other medical imaging AI models by generating diverse and realistic training data.
```

Comment:
Do we need to quote NVIDIA here? @daguangxu ?

Reply:
After confirmed with @daguangxu offline, I removed the quote in the latest commit, thanks!